### PR TITLE
SAK-48115: Trinity Portal: Add Quick Links Button

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-48115

@ern I want the below Sakai properties to be added for testing purposes on nightly server 23: 
**update:**  I have created a pr on sakai-nighlty branch :)
`portal.quicklink.info=Links open in a new window 
portal.quicklink.url.count=4
portal.quicklink.title.count=4
portal.quicklink.name.count=4
portal.quicklink.icon.count=4
portal.quicklink.url.1=https://www.sakailms.org/
portal.quicklink.title.1=Sakai Project Website
portal.quicklink.name.1=Sakai Project
portal.quicklink.icon.1=cl bi bi-globe
portal.quicklink.url.2=https://www.apereo.org/
portal.quicklink.title.2=Apereo Website
portal.quicklink.name.2=Apereo
portal.quicklink.icon.2=cl bi bi-globe
portal.quicklink.url.3=https://github.com/sakaiproject/sakai
portal.quicklink.title.3=Sakai Github
portal.quicklink.name.3=Sakai Github
portal.quicklink.icon.3=cl bi bi-github
portal.quicklink.url.4=https://www.sakailms.org/shop
portal.quicklink.title.4=Sakai Merch Store
portal.quicklink.name.4=Sakai Merch Store
portal.quicklink.icon.4=cl bi bi-cart`

Thanks

### CODE REVIEW

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeAccountNav.vm [hunk 1]
```diff
@@ -54,12 +54,14 @@ For opening help sidebar
...
-          <path d="M4.715 6.542 3.343 7.914a3 3 0 1 0 4.243 4.243l1.828-1.829A3 3 0 0 0 8.586 5.5L8 6.086a1.002 1.002 0 0 0-.154.199 2 2 0 0 1 .861 3.337L6.88 11.45a2 2 0 1 1-2.83-2.83l.793-.792a4.018 4.018 0 0 1-.128-1.287z"/>
-          <path d="M6.586 4.672A3 3 0 0 0 7.414 9.5l.775-.776a2 2 0 0 1-.896-3.346L9.12 3.55a2 2 0 1 1 2.83 2.83l-.793.792c.112.42.155.855.128 1.287l1.372-1.372a3 3 0 1 0-4.243-4.243L6.586 4.672z"/>
-        </svg>
-      </a>
+      <button class="btn icon-button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#sakai-quickLinksPanel"
+          aria-controls="sakai-quickLinksPanel"
+          aria-label="$rloader.sit_quick_links"
+          title="$rloader.sit_quick_links">
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeAccountNav.vm:62]:
```suggestion
          aria-expanded="false"
          title="$rloader.sit_quick_links">
```

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm [hunk 2]
```diff
@@ -55,6 +55,19 @@
...
     #end
 
+  #if ($quickLinks)
+    <li>
+      <button class="btn icon-button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#sakai-quickLinksPanel"
+          aria-controls="sakai-quickLinksPanel"
+          aria-label="$rloader.sit_quick_links"
+          title="$rloader.sit_quick_links">
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm:65]:
```suggestion
          aria-expanded="false"
          title="$rloader.sit_quick_links">
```

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm [hunk 3]
```diff
@@ -55,6 +55,19 @@
...
+        <i class="bi-link-45deg"></i>
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm:67]: I would add an entry to library/src/skins/default/src/sass/base/_icons.scss to decouple this.  Add line under locked in that file.

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Review by adrianfish — APPROVED** [review]
Nice job Kunal

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm (hunk 3 above)
Comment (kunaljaykam) [reply portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeMobileFooter.vm]: alright :)

**Pull request merged**